cmd/cli/namespace_add.go

```diff
@@ -16,11 +16,22 @@ import (
 )
 
 const namespaceAddDescription = `
-This command will add a namespace or a set of namespaces
-to the mesh. Optionally, the namespaces can be configured
-for automatic sidecar injection which enables pods in the
+This command will add a namespace or a set of namespaces 
+to the mesh, automatic sidecar injection is disabled by 
+default.Optionally, the namespaces can be configured for 
+automatic sidecar injection which enables pods in the 
 added namespaces to be injected with a sidecar upon creation.
 `
+const namespaceAddExample = `
+# Add bookstore to the mesh,disabled automatic sidecar injection.If it has been enabled, it will not be disabled.
+osm namespace add bookstore
+
+# Enable automatic sidecar injection
+osm namespace add bookstore --enable-sidecar-injection 
+osm namespace add bookstore --enable-sidecar-injection=true
+
+# Disable automatic  sidecar injection
+osm namespace add bookstore --enable-sidecar-injection=false`
```

Comment:
```suggestion
# Add namespace 'test' to the mesh without enabling automatic sidecar injection. If sidecar injection was previously enabled, it will not be disabled by this command.
osm namespace add test

# Add namespace 'test' to the mesh while enabling automatic sidecar injection
osm namespace add test --enable-sidecar-injection 
osm namespace add test --enable-sidecar-injection=true

# Add namespace 'test' to the mesh while disabling automatic sidecar injection. If sidecar injection was previously enabled, it will be disabled by this command.
osm namespace add test --enable-sidecar-injection=false
```